cdap-storage-spi/src/main/java/io/cdap/cdap/spi/data/StructuredTable.java

```diff
@@ -141,7 +141,7 @@
    *                               do not match
    * @throws IOException if there is an error scanning the table
    */
-  CloseableIterator<StructuredRow> multiScan(Collection<Range> keyRanges,
+  CloseableIterator<StructuredRow>  multiScan(Collection<Range> keyRanges,
```

Comment:
seems like this is an accidental extra space?

Reply:
thanks for pointing that out, reverted.